templates/templates.go

```diff
@@ -178,6 +185,7 @@ func SearchAndFill(toScan string, fileExt string, deleteFile bool, inPlace bool)
 	}
 
 	for _, file := range files {
+		zap.S().Debug("Templating file: " + file)
```

Comment:
maybe `Debugf` will be better here
https://pkg.go.dev/go.uber.org/zap#SugaredLogger.Debugf